### ISSUE TITLE
[Bug]: Videos don't open in Theatre Mode even if "Enable Theatre Mode by Default" is checked

### ISSUE THREAD

**Issue by PrivacyFriendlyMuffins** [FreeTubeApp/FreeTube#5252]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

Even if `Settings > Player Settings > Enable Theatre Mode by Default` is enabled ...

![Freetube Settings](https://github.com/FreeTubeApp/FreeTube/assets/86497467/a6c4fcfb-ef51-48c0-abdd-6d12f3431dfb)

... videos will still open in regular view mode.

![Freetube Regular Mode](https://github.com/FreeTubeApp/FreeTube/assets/86497467/a39f2a6b-7356-461c-928a-9c16585521b6)

(Clicking the Theatre Mode icon in the video toolbar will still successfully toggle between the two states, though.)

### Expected Behavior

When "Enable Theatre Mode by Default" under Settings > Player Settings is set to On, videos should open in Theatre Mode.

![Freetube Theatre Mode](https://github.com/FreeTubeApp/FreeTube/assets/86497467/365ad3d6-cb13-41fc-9baa-d3335f2d90ed)

### Issue Labels

feature stopped working

### FreeTube Version

v0.20.0 Beta

### Operating System Version

Windows 10 Home version 22H2

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.19.2 Beta

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Unable to replicate this on the `master` branch and on my locally installed nightly build

**Comment by PrivacyFriendlyMuffins** [comment]
>     1. Open up the dev tools: `ctrl+shift+i`

Done.

>     2. Show that the setting is enabled

Okay, how do I do that?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Apologies forgot to mention that you need to provide a screen recording of those steps

**Comment by PrivacyFriendlyMuffins** [comment]
Sorry for the late reply. Work and weird hours, you know how it goes. Lol.

Just so I understand you correctly, do you want a screen recording or a screen shot? If a screen recording, I don't know of any software that can do that. Sorry. :( If a screenshot, then no problem; it's above in my original post.

**Comment by jasonhenriquez** [comment]
@PrivacyFriendlyMuffins [Here's an article](https://www.howtogeek.com/706124/how-to-use-windows-10s-built-in-screen-capture-tool/) I found on how to do that in Windows 10, if it helps!

**Comment by PrivacyFriendlyMuffins** [comment]
Thank you!

That being said, did you want a screen recording of the setting being turned on or "something something something dev tools"? Because once I pressed `shift+ctrl+I`, I didn't know what to do to make sure the setting was on. Lol.

**Comment by PhilJbt** [comment]
@PrivacyFriendlyMuffins does this occur only on playlists, like me? I noticed this for a while now, I thought it was just not implemented for playlist, but only for regular watched videos.

Actually on `v0.21.0 Beta`:
[bundle-1718953864154.log](https://github.com/user-attachments/files/15924170/bundle-1718953864154.log)

[2024-06-21 09-10-31.webm](https://github.com/FreeTubeApp/FreeTube/assets/22506952/0449bb36-4f14-458f-af42-3714e98994fd)

**Comment by kelvin-wong** [comment]
I have the same issue.
It could be reproducible by enabling `Hide Recommended Videos` in `Distraction Free Settings`.

**Comment by PhilJbt** [comment]
> I have the same issue. It could be reproducible by enabling `Hide Recommended Videos` in `Distraction Free Settings`.

I can confirm, it was enabled from my side. Disable it fixes the bug.

**Comment by PrivacyFriendlyMuffins** [comment]
> @PrivacyFriendlyMuffins does this occur only on playlists, like me? I noticed this for a while now, I thought it was just not implemented for playlist, but only for regular watched videos.

Yes, I hadn't noticed at the time, but checking that, yes it does seem to only happen on Playlists. If I watch a video from my Subscriptions section, it works just fine.

> > I have the same issue. It could be reproducible by enabling `Hide Recommended Videos` in `Distraction Free Settings`.
> 
> I can confirm, it was enabled from my side. Disable it fixes the bug.

Can confirm this fixes the bug. Unfortunately, that means seeing recommended videos. Lol.

**Comment by loopernow** [comment]
I don't have any Distraction Free settings enabled, but I still experience this behavior--Enable Theatre Mode by Default is on, but videos default to playing in windowed mode.

I'm on Windows 10 using 0.23.3 beta.

**Comment by deepspaceaxolotl** [comment]
Same issue here with `Hide Recommended Videos` enabled, no issue when disabled.
